docs/plugin_developers_guide/09_packaging.rst

```diff
@@ -0,0 +1,256 @@
+.. Documentation of package for a Plugin
+
+Packaging Documentation
```

Comment:
_I don't think we need the word "Documentation" in the title, it is part of the documentation after all._